src/lib/draw-and-pick.js

```diff
@@ -87,27 +87,53 @@ export function pickLayers(gl, {
     });
 
     // Read color in the central pixel, to be mapped with picking colors
-    const color = new Uint8Array(4);
-    gl.readPixels(deviceX, deviceY, 1, 1, GL.RGBA, GL.UNSIGNED_BYTE, color);
+    const pickedColor = new Uint8Array(4);
+    gl.readPixels(deviceX, deviceY, 1, 1, GL.RGBA, GL.UNSIGNED_BYTE, pickedColor);
 
     // restore blend mode
     setBlendMode(gl, oldBlendMode);
     // Picking process end
 
     // Process picked info start
     // Decode alpha to layer index
-    const pickedLayer = layers[color[3] - 1];
+    const pickedLayer = pickedColor[3] ? layers[pickedColor[3] - 1] : null;
```

Comment:
Maybe split in two lines to make more self documenting:
```
const pickedLayerIndex  = pickedColor[3] - 1;
const pickedLayer = pickedLayerIndex > -1  ? layers[pickedLayerIndex] : null;
```

Side note: I've also considered making the background picking color white (0xFFFFFF) instead of black, so that we don't have to add and subtract one to the picking colors. Would make them a little more intuitive